registry/config/config.ini

```diff
@@ -31,7 +31,7 @@ api_otp=True
 cors=*
 
 # Set dev enviroment off ie. https checks are on
-dev=False
+dev=True

```

Comment:
please don't modify default config, this is deployment specific and can be overwritten